lib/services.rb

```diff
@@ -1,4 +1,5 @@
 require "gds_api/asset_manager"
+require "gds_api/content_store"
```

Comment:
I now realise that a `Services.content_store` method already makes the Content Store available. Thus I think my last comment isn't quite right. I think it would be clearer to amend the commit note to say that `lib/services.rb` should explicitly require all its dependencies. I think mentioning the Rails console is confusing.